presto-orc/src/main/java/com/facebook/presto/orc/OrcWriter.java

```diff
@@ -637,7 +620,7 @@ private void flushColumnWriters(FlushReason flushReason)
         closedStripesRetainedBytes += closedStripe.getRetainedSizeInBytes();
 
         recordValidation(validation -> validation.addStripe(stripeInformation.getNumberOfRows()));
-        stats.recordStripeWritten(stripeMinBytes, stripeMaxBytes, dictionaryMaxMemoryBytes, flushReason, dictionaryCompressionOptimizer.getDictionaryMemoryBytes(), stripeInformation);
```

Comment:
stripeMinBytes, stripeMaxBytes, dictionaryMaxMemoryBytes values were not used by stats, so I removed them.

Reply:
Let us keep passing this value. We need to record this for Jolly when our stats implementation is ready.